storybook/src/components/FieldSet/FieldSet.docs.mdx

```diff
@@ -24,7 +24,15 @@ and provide the `id` of the describing element as its value.
 
 <Canvas of={FieldSetStories.WithDescription} />
 
-## With Error
+### With Not Required Label
+
+A Field Set can indicate whether all of the inputs it contains are <em>not</em> required.
+Make sure to <em>not</em> add a `required` or `aria-required="true"` attribute to <em>any</em> form element in this Field Set.
+Also never set either the `aria-required` attribute on a field set that is not of the `role="radiogroup"` (see: [Radio Group](#radio-group)).
```

Comment:
Not sure what this must convey; can you rephrase?

Reply:
Removed the sentence and changed the text at the Radio group story.